src/screens/products/components/desmos/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { useTranslation } from "i18n";
+import { DesmosContentCSS } from "./styles";
+import { Go } from "@icons";
+
+export const DesmosContent = () => {
+  const { t } = useTranslation("products");
+  return (
+    <DesmosContentCSS>
+      <div className="desktopWrapper">
+        <div className="image">
+          <img
+            src="/static/images/assets/desmos-image.png"
+            alt="Forbole Portal Screenshot"
+          ></img>
+        </div>
+        <div className="content">
+          <h1>{t("desmos")}</h1>
+          <li>{t("dd1")}</li>
+          <li>{t("dd2")}</li>
+          <li>{t("dd3")}</li>
+          <a href="https://www.desmos.network/">
```

Comment:
If this is one of the list items, the `<a>` should be a child of `<li>`.
```suggestion
          <li><a href="https://www.desmos.network/">
```

Reply:
@kwunyeung this is the `EXPLORER -->` under the list items
![image](https://user-images.githubusercontent.com/53459742/111655168-62b33f00-8844-11eb-932f-b2b23d4454c2.png)

Reply:
Then the `<a>` should be a sibling of the `<ul>`.